Meridian59/Data/Models/AdminData/AdminData.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Meridian59.Common.Interfaces;
+using Meridian59.Data.Lists;
+using Meridian59.Protocol.Events;
+using Meridian59.Protocol.GameMessages;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminData : INotifyPropertyChanged, IClearable
+    {
+        //Handles the objects we're watching in the AdminUI
+        private BaseList<AdminObject> TrackedObjects { get; set; }
+        public void TrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Add(adminObject);
+            OnTrackedAdminObjectAdded(new AddTrackedAdminObjectEventHandlerArgs(this, adminObject));
+        }
+        public void UntrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Remove(adminObject);
+        }
+
+        public AdminData()
+        {
+            TrackedObjects = new BaseList<AdminObject>();
+        }
+
+        /// <summary>
+        /// Interface to the Client's Network
+        /// </summary>
+        public event GameMessageEventHandler PacketSend;
+
+        #region events
+        //Event to add an object 
+        public delegate void AddTrackedAdminObjectEventHandler(object sender, AddTrackedAdminObjectEventHandlerArgs args);
+        public event AddTrackedAdminObjectEventHandler AdminObjectAdded;
+
+        protected void OnTrackedAdminObjectAdded(AddTrackedAdminObjectEventHandlerArgs e)
+        {
+            if (AdminObjectAdded != null) AdminObjectAdded(this, e);
+        }
+
+        //Event to pass a message back to the console
+        public delegate void LogAdminMessageEventHandler(LogAdminMessageEventHandlerArgs e);
+        public event LogAdminMessageEventHandler LogAdminMessage;
+        protected void OnLogAdminMessage(LogAdminMessageEventHandlerArgs e)
+        {
+            if (LogAdminMessage != null)
+            {
+                LogAdminMessage(e);
+            }
+        }
+        #endregion
+
+        void adminObjectProperty_ValueChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (sender.GetType() == typeof (AdminObjectProperty))
```

Comment:
I'm usually using the "is" keyword to check class, however there is a difference to your variant here.
This expression will probably return "false" for a class deriving from 'AdminObjectProperty'.
If this is what you want, it's OK. But usually you would "accept" any "more specialized" class too.
So this could be:
>> if (sender is AdminObjectProperty)